prime-router/build.gradle.kts

```diff
@@ -131,6 +133,8 @@ tasks.clean {
  */
 val coverageExcludedClasses = listOf("gov/cdc/prime/router/azure/db/*", "gov/cdc/prime/router/cli/tests/*")
 tasks.test {
+    group = rootProject.description ?: ""
+    description = "Run the unit tests"
```

Comment:
speaking of tests... hiding large tables (really) should affect at least unit tests.  What is a good way to communicate this process (hiding large tables) to writers of future tests?  Is there a "good practice?"

Reply:
Well, the CLI routines do not have unit tests.  As far as communication, all CLI commands use the --help for all options.